linters/remark-lint/plugin.yaml

```diff
@@ -0,0 +1,45 @@
+version: 0.1
+lint:
+  definitions:
+    - name: remark-lint
+      files: [markdown]
+      package: remark-cli
+      extra_packages:
+        - remark-preset-lint-consistent
+        - remark-preset-lint-recommended
+        - remark-lint-list-item-indent
+        - remark-lint-no-tabs
+        - vfile-reporter-json
+      runtime: node
+      known_good_version: 11.0.0
+      direct_configs:
+        - .remarkrc
+        - .remarkrc.json
+        - .remarkrc.cjs
+        - .remarkrc.mjs
+        - .remarkrc.js
+        - .remarkrc.yaml
+        - .remarkrc.yml
```

Comment:
no hidden .editorconfig, right? I know a lot of node ones have it as undocumented.

Reply:
Not that it specified.